packages/reket-core/tests/config/item.spec.js

```diff
@@ -0,0 +1,16 @@
+import { ReketConfigItem } from '../../src/config/item';
+
+describe('ReketConfigItem implementation', () => {
+  test('it should instanciate a ReketConfigItem instance', () => {
+    const itemConfig = new ReketConfigItem();
+
+    expect(itemConfig.value).toBe(undefined);
+  });
+
+  test('it should set a value to the config item', () => {
+    const itemConfig = new ReketConfigItem();
+    itemConfig.set('a value');
+
+    expect(itemConfig.value).toBe('a value');
+  });
```

Comment:
In the set of ReketConfigItem, we're just doing "this.#value = value" and there's no need to test that. 

In the ReketConfigClient tests, it was relevant to test the setter, because the setter had logic that threw error if the type was incorrect and we wanted to ensure that that was working correctly. In this case, there is no test, it's just a plain instantiation and it's the same as before, it's just js, there's not logic to test. 😄